doc/input-output-reference/src/overview/group-advanced-surface-concepts.tex

```diff
@@ -1,4 +1,10 @@
-\section{Group -- Advanced Surface Concepts}\label{group-advanced-surface-concepts}
+\documentclass[10pt]{article}
```

Comment:
@RKStrand This standalone LaTeX preamble will have to be removed back out so that the documentation can be built as part of the full pdf.